src/main/java/ca/uhn/fhir/jpa/starter/controller/DashboardController.java

```diff
@@ -50,9 +50,13 @@ public void init() {
 
 		@RequestMapping(method = RequestMethod.GET, value = "/details")
 	public ResponseEntity<?> getDetails(
+		@RequestHeader(name = "Authorization") String token,
 		@RequestParam("env") String env,
 		@RequestParam Map<String, String> allFilters
 	) throws SQLException, IOException {
+		Boolean isAnonymizationEnabled = Validation.getJWTToken(token).getAnonymization();
+		if (isAnonymizationEnabled == null)
+			isAnonymizationEnabled = false;
```

Comment:
this check is repeated at every function, make getAnonymization() return false.

Reply:
Creating a Strategy pattern and extracting this to it.